src/diffusers/models/adapter.py

```diff
@@ -152,6 +154,48 @@ def __init__(
 
         self.total_downscale_factor = downscale_factor * 2 ** (len(channels) - 1)
 
+    def forward(self, x: torch.Tensor) -> List[torch.Tensor]:
+        x = self.unshuffle(x)
+        x = self.conv_in(x)
+
+        features = []
+
+        for block in self.body:
+            x = block(x)
+            features.append(x)
+        
+        return features
```

Comment:
Not sure why it's shown as a change as it's same with `main`

https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/adapter.py#L155C1-L165C24